### PR TITLE
[Tests-Only] Add webUI tests for overwriting a file while upload is in progress

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Add webUI tests for uploading a file and overwriting when the upload is in progress

## Related Issue

- Related https://github.com/owncloud/ocis/issues/1079

## How Has This Been Tested?

🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/stepDefinitions/generalContext.js [hunk 1]
```diff
@@ -103,6 +118,16 @@ Then('the following success/error message should be displayed on the webUI', fun
     .text.to.equal(message)
 })
 
+Then('a error message should be displayed on the webUI containing following text', function(
```
Comment (individual-it) [inline tests/acceptance/stepDefinitions/generalContext.js:121]: duplicate of the function above
for discussion see also https://github.com/owncloud/web/pull/4538

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/webUIUpload/uploadEdgecases.feature [hunk 2]
```diff
@@ -128,3 +128,19 @@ Feature: File Upload
     #   """
     #   (Any nice error message)
     #   """
+
+  Scenario Outline: Overwrite a file when upload is in progress
```
Comment (individual-it) [inline tests/acceptance/features/webUIUpload/uploadEdgecases.feature:132]: tag the Scenario with the issue and add steps that show the correct behaviour

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/stepDefinitions/generalContext.js (hunk 1 above)
Comment (dpakach) [reply tests/acceptance/stepDefinitions/generalContext.js:121]: not actually, it checks if the message contains given text

File: tests/acceptance/features/webUIUpload/uploadEdgecases.feature (hunk 2 above)
Comment (dpakach) [reply tests/acceptance/features/webUIUpload/uploadEdgecases.feature:132]: I could not reporduce the issue here, I think this is the correct behavior

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Upload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12535/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISUpload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12535/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Upload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12645/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISUpload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12645/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Upload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12650/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISUpload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12650/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Upload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12651/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISUpload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12651/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISUpload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12666/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Upload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12674/

<p>










</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISUpload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12674/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Upload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12675/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISUpload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12675/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISUpload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12679/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISUpload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/12704/

<p>





</p></details>

**Comment by dpakach** [comment]
I tried having an automated test for https://github.com/owncloud/ocis/issues/1079, but could not get it to work in drone. while the test work mostly on my local system, it fails intermittently. It may be because of the difference in system/upload speed. So I think it will be more beneficial to just test this issue manually rather than having the automated test.

**Pull request closed without merging**